templates/about.hbs

```diff
@@ -123,7 +123,7 @@ dependencies = [ "example-system-dependency" ]</pre></code>
   <p>Currently running Docs.rs version is: <strong>{{cratesfyi_version}}</strong>
 
   <h4>Contact</h4>
-  <p>Docs.rs is run and maintaned by <a href="https://www.rust-lang.org/governance/teams/dev-tools#Rustdoc%20team-info" target="_blank">Rustdoc team</a>. You can find us in #rustdoc on <a href="https://discord.gg/rust-lang" target="_blank">Discord</a>.</p>
+  <p>Docs.rs is run and maintaned by <a href="https://www.rust-lang.org/governance/teams/dev-tools#rustdoc" target="_blank">Rustdoc team</a>. You can find us in #rustdoc on <a href="https://discord.gg/rust-lang" target="_blank">Discord</a>.</p>
```

Comment:
While we're here, can we fix the discord channel to `#docs-rs`? Since it's got its own channel, we might as well point to that one.